V/VMEC/build_tarballs.jl

```diff
@@ -0,0 +1,45 @@
+using BinaryBuilder
+
+name = "VMEC"
+version = v"1.0.0"
+
+sources = [
+    ArchiveSource("https://gitlab.com/wistell/VMEC2000/-/archive/v1.0.0.tar", "179761d5f02cb530fbf6da4121c394a1d585c49c24391ebd87fe5c014e0addcf"),
+]
+
+script = raw"""
+mkdir -p ${libdir}
+cd ${WORKSPACE}/srcdir/VMEC2000*
+make USE_MKL=1
+install -m755 lib/libvmec_mkl.so ${libdir}
+make USE_MKL=1 vmec_clean ; make USE_MKL=0
+install -m755 lib/libvmec_openblas.so ${libdir}
+make USE_MKL=0 vmec_clean
+"""
+
+platforms = [
+    Platform("x86_64", "linux"; libc = "glibc", libgfortran_version="4.0.0"),
```

Comment:
Why there isn't libgfortran 3?

Reply:
The underlying Fortran code doesn't compile with libgfortran3.  Unfortunately this is a rather legacy code for which I am not the author nor primary developer, and determining the changes required is not a great use of my time.  The user base is small and very targeted towards scientific computing/HPC applications where there should not be an issue with having current libgfortran versions.